aws/provider_test.go

```diff
@@ -1173,6 +1174,57 @@ func testAccPreCheckSkipError(err error) bool {
 	return false
 }
 
+type testSweepResource struct {
+	d        *schema.ResourceData
+	meta     interface{}
```

Comment:
Since the idea is to promote to the plugin SDK, I think we leave this as a generic. Not saying this isn't a battle worth fighting but meta is everywhere and always generic.